localstack/aws/handlers/logging.py

```diff
@@ -95,6 +95,10 @@ def _log(self, context: RequestContext, response: Response):
                     response.status_code,
                     context.service_exception.code,
                     extra={
+                        "request_id": context.request_id,
```

Comment:
`request_id` propagation is gonna be crucial for proper grouping and is going to require quite some manual work within service-specific loggers. Other fields can then be inferred based on the `request_id` because some other fields won't be available at different stages (e.g., service, region, and operation won't be available before parsing an HTTP request into an AWS request; response-related fields such as status code obviously won't be available before handling the request)